src/thor/costmatrix.cc

```diff
@@ -6,6 +6,8 @@
 #include "thor/costmatrix.h"
 #include "worker.h"
 
+#include <robin_hood.h>
```

Comment:
I dont think this is a system installed header, in which case its canonical to use `#include "robin_hood.h"` (instead of the angled braces).

Reply:
We use cmake to import thir party header directories essentially with the `-I` flag. The angle brackets are correct because of this